### PR TITLE
Responsive Component doc update

### PR DESCRIPTION
**Pull request by wrpeck**
_Description of changes:_

Updating Responsive Component documentation for the improved features

### CODE REVIEW

File: src/pages/console/uibuilder/responsive.mdx [hunk 1]
```diff
@@ -22,39 +24,89 @@ By default, all Amplify UI provides the following [breakpoints as part of the Am
 }
 ```
 
-Configure your Figma components with variants with names that match the breakpoint values. In the example below, our `TopBar` component has the following variants:
-- `variation: default`
-- `variation: small`
+Breakpoints are set at the [Amplify UI Theme](https://ui.docs.amplify.aws/react/theming/responsive) level, which means they apply to all components in your Figma file.  You can customize these breakpoints using the Amplify Theme Editor plugin:
+1. Right click in your Figma project and select "Plugins" > "AWS Amplify Theme Editor"
```
Comment (rachelnabors) [inline src/pages/console/uibuilder/responsive.mdx:28]: Remove quotes and bold the commands

File: src/pages/console/uibuilder/responsive.mdx [hunk 2]
```diff
@@ -22,39 +24,89 @@ By default, all Amplify UI provides the following [breakpoints as part of the Am
...
+![Context menu showing how to view plugins and select the AWS Amplify Theme Editor](/images/studio/responsive/theme-editor.png)
+
+2. Select the "Breakpoints" Tab and click on any breakpoint to customize the pixel value
```
Comment (rachelnabors) [inline src/pages/console/uibuilder/responsive.mdx:31]: "Tab" should be lowecase

File: src/pages/console/uibuilder/responsive.mdx (hunk 1 above)
Comment (renebrandel) [inline src/pages/console/uibuilder/responsive.mdx:28]: I think you need an empty line above here for the docs to interpret this correctly. (not sure if this is still the case)

File: src/pages/console/uibuilder/responsive.mdx [hunk 3]
```diff
@@ -22,39 +24,89 @@ By default, all Amplify UI provides the following [breakpoints as part of the Am
...
+![AWS Amplify Theme Editor with the Breakpoints tab selected](/images/studio/responsive/breakpoints.png)
+
+The grey bar within the outlined box will indicate your pixel value compared to a standard 1920px FHD resolution.
+
+## 2. Configure your Figma variants
 
-The small variation should be shown when the screen size falls below the "small" breakpoint.
+To make a component responsive, create different variants for the component based on the breakpoints.  [Learn more about creating variants in Figma's documentation](https://help.figma.com/hc/en-us/articles/360061175334-Create-interactive-components-with-variants).
```
Comment (renebrandel) [inline src/pages/console/uibuilder/responsive.mdx:38]:
```suggestion
To make a component responsive, create different variants for the component based on the breakpoints. [Learn more about creating variants in Figma's documentation](https://help.figma.com/hc/en-us/articles/360061175334-Create-interactive-components-with-variants).
```

File: src/pages/console/uibuilder/responsive.mdx [hunk 4]
```diff
@@ -22,39 +24,89 @@ By default, all Amplify UI provides the following [breakpoints as part of the Am
 }
 ```
 
-Configure your Figma components with variants with names that match the breakpoint values. In the example below, our `TopBar` component has the following variants:
-- `variation: default`
-- `variation: small`
+Breakpoints are set at the [Amplify UI Theme](https://ui.docs.amplify.aws/react/theming/responsive) level, which means they apply to all components in your Figma file.  You can customize these breakpoints using the Amplify Theme Editor plugin:
```
Comment (renebrandel) [inline src/pages/console/uibuilder/responsive.mdx:27]:
```suggestion
Breakpoints are set at the [Amplify UI Theme](https://ui.docs.amplify.aws/react/theming/responsive) level, which means they apply to all components in your Figma file. You can customize these breakpoints using the Amplify Theme Editor plugin:
```

File: src/pages/console/uibuilder/responsive.mdx [hunk 5]
```diff
@@ -22,39 +24,89 @@ By default, all Amplify UI provides the following [breakpoints as part of the Am
...
 
-**IMPORTANT:** The variants are required to have the same component structure. If the variants don't have the same child elements, then Amplify Studio will not be able to import the component.
+**IMPORTANT:** The variants are required to have the same <strong>component structure</strong>. Elements within the component structure should use the same <strong>layout method</strong> (i.e. auto or manual). 
 
-![Image showing that the component structure needs to be the same](/images/studio/responsive/component-structure.png)
+If the variants don't have the same structure, then Amplify Studio will not be able to import the component.
+
+To customize child elements in variants, you can use the eye icon to hide and show elements in different variants. 
+
+![Image showing that the component structure needs to be the same](/images/studio/responsive/component-structure2.png)
```
Comment (renebrandel) [inline src/pages/console/uibuilder/responsive.mdx:48]: If you're changing the image, you should maybe just remove the older image. Is the other image used somewhere else?

File: src/pages/console/uibuilder/responsive.mdx [hunk 6]
```diff
@@ -22,39 +24,89 @@ By default, all Amplify UI provides the following [breakpoints as part of the Am
...
 
 </Callout>
 
-## 2. Render component variants in code based on breakpoints
+When generating a component with variants in Figma, a Property will be created and assigned to each variant.  Make the following configurations to your component:
```
Comment (renebrandel) [inline src/pages/console/uibuilder/responsive.mdx:52]:
```suggestion
When generating a component with variants in Figma, a Property will be created and assigned to each variant. Make the following configurations to your component:
```

**Review by rachelnabors — CHANGES_REQUESTED** [review]
A few updates to the as-yet-to-be-published styleguide!

File: src/pages/console/uibuilder/responsive.mdx [hunk 7]
```diff
@@ -22,39 +24,89 @@ By default, all Amplify UI provides the following [breakpoints as part of the Am
...
-import { useBreakpointValue } from '@aws-amplify/ui-react'
+## 3. Render and test your component
 
+With your design complete in Figma, import and render your component using Amplify Studio UI Builder:
+1. [Sync your Figma file with UI Builder](figmatocode/)
+2. In your local project, run `amplify pull`
+3. Import and render your new React component in code
+4. Resize the window - your component will change automatically based on the breakpoints you set before
+
+## 4. Multi-dimensional variants
```
Comment (renebrandel) [inline src/pages/console/uibuilder/responsive.mdx:67]:
```suggestion
## Advanced: Configure responsive components with existing variants
```

File: src/pages/console/uibuilder/responsive.mdx [hunk 8]
```diff
@@ -22,39 +24,89 @@ By default, all Amplify UI provides the following [breakpoints as part of the Am
...
+
+Figma allows users to create variants based on multiple properties.  For example, a component may need `breakpoint` variants (small, medium, and large) and `displaymode` variants (light mode, dark mode).
```
Comment (renebrandel) [inline src/pages/console/uibuilder/responsive.mdx:69]:
```suggestion
Figma allows users to create variants based on multiple properties.  For example, a component may need `breakpoint` variants (small, medium, and large) and `displayMode` variants (light mode, dark mode).
```
nit: but this casing is more idiomatic.

File: src/pages/console/uibuilder/responsive.mdx [hunk 9]
```diff
@@ -22,39 +24,89 @@ By default, all Amplify UI provides the following [breakpoints as part of the Am
...
+
+If your component has any variant properties in addition to `breakpoint`, additional steps are required for responsiveness.  All additional properties <i>must</i> be given inline values, or the component will not be responsive.  
+
+In the below examples, the `ResponsiveToolbar` component has both `breakpoint` and `displaymode` variants.  In the first example, the component will not automatically be responsive.
```
Comment (renebrandel) [inline src/pages/console/uibuilder/responsive.mdx:73]:
```suggestion
In the below examples, the `ResponsiveToolbar` component has both `breakpoint` and `displayMode` variants.  In the first example, the component will not automatically be responsive.
```

File: src/pages/console/uibuilder/responsive.mdx (hunk 5 above)
Comment (wrpeck) [reply src/pages/console/uibuilder/responsive.mdx:50]: Older file has been removed!

File: src/pages/console/uibuilder/responsive.mdx (hunk 1 above)
Comment (wrpeck) [reply src/pages/console/uibuilder/responsive.mdx]: Quote removed and commands bolded

File: src/pages/console/uibuilder/responsive.mdx (hunk 2 above)
Comment (wrpeck) [reply src/pages/console/uibuilder/responsive.mdx]: Tab changed to lowercase

**Review by rachelnabors — APPROVED** [review]

**Review by mauerbac — APPROVED** [review]
LGTM

**Review by jacoblogan — APPROVED** [review]

**Pull request merged**